### DISCUSSION TITLE
2021-09-29 Weekly Meeting 23 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#102] [Weekly Meetings]
Weekly Meeting 23 is happening on Wednesday September 29, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - Call For Participation Issue Template
   - OMI Website
   - Vket Community Booth Planning
   - glTF subgroup
   - OMI Community Development Group

Add ons: 
   - Next virtual world community field trip?
   - ...

**Comment by robertlong** [comment]
Notes:

- Nova to lead a virtual field trip in Neos. Figuring out a time for that. Robert and Evo to help organize.
- Work on glTF Validator for OMI_audio_emitter before meeting tomorrow
- Need for OMI Open House-type event for new members to meet each other and possibly summarize/recap some of our long term work.
